templates/_meetings.tt

```diff
@@ -0,0 +1,5 @@
+{% macro render(meetings, empty="No other meetings scheduled.") %}
+{%- for mtg in meetings %}
+- [{{ mtg.summary }}]({{ mtg.html_link }}) at <time:{{ mtg.start.date_time }}>{% else %}
+- {{empty}}{% endfor -%}
+{% endmacro %}
```

Comment:
each meeting will be printed in a markdown format that Zulip will like, example:
```
### Other WG meetings ([calendar link](https://calendar.google.com/calendar/embed?src=6u5rrtce6lrtv07pfi3damgjus%40group.calendar.google.com))- next weeks' meetings [from google calendar](https://calendar.google.com/calendar/htmlembed?src=6u5rrtce6lrtv07pfi3damgjus%40group.calendar.google.com)

- [wg-rls-2.0 weekly sync-up](https://www.google.com/calendar/event?eid=MXJnbzBscDExNHJjNmVsdGhsaW8xcDljMmdfMjAyMjA2MTNUMTUwMDAwWiA2dTVycnRjZTZscnR2MDdwZmkzZGFtZ2p1c0Bn) at <time:2022-06-13T11:00:00-04:00>
- [Performance Triage](https://www.google.com/calendar/event?eid=NG1qdG9tYWs1ZmJmYmwydnFqYWcwNjY4MmNfMjAyMjA2MTRUMTUwMDAwWiA2dTVycnRjZTZscnR2MDdwZmkzZGFtZ2p1c0Bn) at <time:2022-06-14T11:00:00-04:00>
- [Types Team: Planning/Deep-Dive meeting](https://www.google.com/calendar/event?eid=M3JsYmdkN2F0cjdtYXE0Z2ljYzM4Nm42NTRfMjAyMjA2MTdUMTMwMDAwWiA2dTVycnRjZTZscnR2MDdwZmkzZGFtZ2p1c0Bn) at <time:2022-06-17T09:00:00-04:00>
```